frameworks/Dart/stream/stream-nginx.dockerfile

```diff
@@ -2,6 +2,7 @@ FROM tfb/nginx:latest
 
 COPY ./ ./
 
+RUN chmod a+rwx nginx-conf.sh
```

Comment:
Hey man... don't add. Now we won't know if this line fixes it or all my insanity in the `nginx.conf`